app/assets/stylesheets/common/components/hashtag.scss

```diff
@@ -58,7 +58,7 @@ a.hashtag {
 .hashtag-autocomplete {
   max-height: 13.5em;
   overflow-y: auto;
-  box-shadow: shadow("dropdown-lite");
+  box-shadow: var(--shadow-dropdown);
```

Comment:
One variant less 🚀

Reply:
I used it in chat and the new hashtag dropdowns. I find the difference with the regular shadow-dropdown pretty harsh. It's much too strong